Upstream-Status: Backport

* Backported from the ToT: https://crrev.com/c/1280643

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From b27cd1f669000c388d7ee3ea74de629df053571c Mon Sep 17 00:00:00 2001
From: Jaeyong Bae <jdragon.bae@gmail.com>
Date: Wed, 17 Oct 2018 23:27:15 +0000
Subject: [PATCH 05/17] [ozone/wayland] Clean up duplicated header

platform_screen.h is included in wayland_screen.h.
It would be duplicated and we can remove it.

Change-Id: I14cb3c03e28c7ebc79c579eaac238325c670448d
Reviewed-on: https://chromium-review.googlesource.com/c/1280643
Reviewed-by: Maksim Sisov <msisov@igalia.com>
Commit-Queue: Jaeyong Bae <jdragon.bae@gmail.com>
Cr-Commit-Position: refs/heads/master@{#600595}
---
 ui/ozone/platform/wayland/ozone_platform_wayland.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ui/ozone/platform/wayland/ozone_platform_wayland.cc b/ui/ozone/platform/wayland/ozone_platform_wayland.cc
index ddddd8615275..642a276b0c12 100644
--- a/ui/ozone/platform/wayland/ozone_platform_wayland.cc
+++ b/ui/ozone/platform/wayland/ozone_platform_wayland.cc
@@ -21,7 +21,6 @@
 #include "ui/ozone/public/gpu_platform_support_host.h"
 #include "ui/ozone/public/input_controller.h"
 #include "ui/ozone/public/ozone_platform.h"
-#include "ui/ozone/public/platform_screen.h"
 #include "ui/platform_window/platform_window_init_properties.h"
 
 #if BUILDFLAG(USE_XKBCOMMON)
-- 
2.17.1

